test_utils/fake_course_discovery_api.py

```diff
@@ -15,7 +15,33 @@
     "status": "active",
```

Comment:
Not currently utilizing the changes below, but likely will when I write the tests.